contracts/GolemNetworkTokenWrapped.sol

```diff
@@ -35,6 +36,16 @@ contract BatchToken is StandardToken {
 
         balances[msg.sender] = balance;
     }
+
+    function transferAndCall(address to, uint256 value, bytes data) external returns (bool) {
```

Comment:
Because we are not following any ERC directly, this function should no return any value. Always returning `true` does not have much sense.

Reply:
Good point.